### ISSUE TITLE
runtime.txt won't install other python 3 versions

### ISSUE THREAD

**Issue by choldgraf** [jupyter/repo2docker#146]
Surfaced by @foozmeat in the gitter channel, if you put something like `python-3.6` then it doesn't look like the runtime gets updated. I don't see anything here:

https://github.com/jupyter/repo2docker/blob/master/repo2docker/detectors.py#L505

that would install other versions of python 3...what would be the best way to go about this?

**Comment by yuvipanda** [comment]
Yep, we don't support python3.6 and should.

This should be a modification in the Python build pack that decides which packages to include.

**Comment by choldgraf** [comment]
do you think this should replace the 3.5 kernel with 3.6, or install it alongside 3.5 similar to how 2.7 behaves?

**Comment by yuvipanda** [comment]
definitely replace. Don't think we can / should support multiple 3.x kernels at the same time for now.

**Comment by choldgraf** [comment]
hmmm - so it looks like the default in apt is python3.5 yeah? would we need to pull it from a different repo?

**Comment by choldgraf** [comment]
then again, we could always..................handle this with minicoda _runs away_

**Comment by yuvipanda** [comment]
there's a python3.6 package in zesty so that should work. we can also switch to artful from zesty (new versions of everything!) but perhaps we should decouple these two into different PRs.

**Comment by yuvipanda** [comment]
@choldgraf you could, by using environment.yml :) Also let's keep the (good natured) conda/venv related trolling to a minimum please :)

**Comment by choldgraf** [comment]
Actually now that I think about it I'm a bit confused...the person that
noticed this originally mentioned that they tried using environment.yml to
define the environment but that it then skipped requirements.txt . To my
knowledge that should have worked fine no? I feel like there's another
issue that addresses this but I'm on a plane/on my phone so can't
investigate

**Comment by foozmeat** [comment]
Hi, it was me that ran into the issue. I first tried using a `runtime.txt` and that got me python 3.5 even though I specified 3.6. When I tried an `environment.yml` with just the runtime specified then my `requirements.txt` was skipped. I finally added my core packages to the `environment.yml` and now its working fine. I added test code to the top of my notebook to check for python 3.6 and it succeeds

https://hub.mybinder.org/user/panicinc-speed_mystery-eyd96te7/notebooks/main.ipynb
